extensions/servlet/test/com/google/inject/servlet/ServletDefinitionPathsTest.java

```diff
@@ -93,6 +93,12 @@ protected void service(HttpServletRequest servletRequest,
     expect(request.getServletPath())
         .andReturn(requestPath);
 
+    // computePath()
```

Comment:
rm rather than commenting out.  but my main question here is: why is this no longer necessary?